stock_pull_list/wizards/stock_pull_list_wizard.xml

```diff
@@ -0,0 +1,131 @@
+        <field name="arch" type="xml">
+            <form>
+                <group string="Needs">
+                    <field name="line_ids" readonly="1" nolabel="1">
+                        <tree>
+                            <field name="wizard_id" invisible="1" />
+                            <field name="product_id" />
+                            <field name="location_id" />
+                            <field name="date" />
+                            <field name="available_qty" />
+                            <field name="incoming_qty" />
+                            <field name="raw_demand_qty" />
+                            <field name="needed_qty" />
+                            <field name="stock_rule_id" />
+                            <field name="selected" widget="toggle_button" />
+                        </tree>
+                    </field>
+                </group>
+                <group string="Filter Selected">
+                    <group>
+                        <field name="consolidate_by_product" invisible="1" />
+                        <field name="select_all" />
+                        <field
+                            name="rule_action"
+                            attrs="{'invisible':[('select_all', '!=', False)]}"
+                        />
+                        <field
+                            name="available_in_source_location"
+                            attrs="{'invisible':['|', ('consolidate_by_product', '!=', True), ('select_all', '!=', False)]}"
+                        />
+                        <field
+                            name="exclude_reserved"
+                            attrs="{'invisible':['|', ('available_in_source_location', '!=', True), ('select_all', '!=', False)]}"
+                        />
+                    </group>
+                    <group>
+                        <button
+                            name="action_update_selected"
+                            string="Apply Filter"
+                            type="object"
+                            icon="fa-cogs"
+                        />
+                    </group>
+                </group>
+                <group name="grouping" string="Split/Grouping Options">
+                    <group>
+                        <field name="max_lines" />
+                    </group>
+                    <group>
+                        <field name="group_by_rule" />
+                    </group>
+                </group>
+                <footer>
+                    <button
+                        name="action_procure"
+                        string="Procure"
+                        type="object"
+                        class="oe_highlight"
+                    />
+                    or
+                    <button string="Cancel" class="oe_link" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+    <act_window
```

Comment:
act_window is deprecated covert this to a `<record ... model="ir.actions.act_window">...`